pkg/exasol-rest-api/application_properties.go

```diff
@@ -1,48 +1,61 @@
 package exasol_rest_api
 
 import (
-	"errors"
+	"fmt"
+	error_reporting_go "github.com/exasol/error-reporting-go"
 	"os"
 )
 
-//ApplicationProperties for Exasol REST API service.
+// ApplicationProperties for Exasol REST API service.
 type ApplicationProperties struct {
 	ApplicationServer         string `yaml:"server-address"`
 	ExasolUser                string `yaml:"exasol-user"`
 	ExasolPassword            string `yaml:"exasol-password"`
 	ExasolHost                string `yaml:"exasol-host"`
 	ExasolPort                int    `yaml:"exasol-port"`
-	ExasolWebsocketApiVersion int    `yaml:"exasol-websocket-api-version"`
+	ExasolWebsocketAPIVersion int    `yaml:"exasol-websocket-api-version"`
 	Encryption                bool   `yaml:"encryption"`
 	UseTLS                    bool   `yaml:"use-tls"`
 }
 
-//GetApplicationProperties creates an application properties.
+// GetApplicationProperties creates an application properties.
 func GetApplicationProperties(applicationPropertiesPathKey string) *ApplicationProperties {
 	propertiesPath := os.Getenv(applicationPropertiesPathKey)
 	if propertiesPath == "" {
-		panic("runtime error: missing environment variable: " + applicationPropertiesPathKey)
+		panic(error_reporting_go.ExaError("E-ERA-4").Message("runtime error: missing environment variable: {{env}}.").
```

Comment:
Why `runtime error:`?

I did not understand the concept why it's there in some messages and in others not.
Maybe just leave it away?

Reply:
And start in uppercase

Reply:
Because it's panic. Other errors are just errors. 
And the lowercase in error is a standard in Go